src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -158,7 +146,7 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $inter
       if (animateTo === animateFrom) {
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
-        (function animation() {
+        $$rAF(function animation() {
```

Comment:
Oh, I though that before you were referring to the case a little bit above where `animateTo === animateFrom`.